apps/api/README.md

```
@@ -0,0 +1,8 @@
+```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)